website/project/model.py

```diff
@@ -1689,10 +1689,14 @@ def save(self, *args, **kwargs):
         if 'node_license' in saved_fields:
             children = [c for c in self.get_descendants_recursive(
                 include=lambda n: n.node_license is None
-            )]
+            ) if c.is_public and not c.is_deleted]
             # this returns generator, that would get unspooled anyways
             if children:
-                Node.bulk_update_search(children)
+                if len(children) > 100:
+                    Node.bulk_update_search(children[:99])
+                    Node.bulk_update_search(children[99:])
```

Comment:
We won't run into the problem for a while, but since we're solving this anyways, let's make this handle an arbitrary number of nodes. Perhaps a loop?